how-to-ssr/public/javascripts/main.js

```diff
@@ -0,0 +1,11 @@
+requirejs.config({
+  paths: {
+    jquery: 'https://code.jquery.com/jquery-3.2.1.min',
+    'chegg-oc-client': 'http://components-test-cdn.cheggnet.com/components/oc-client/0.36.27/src/oc-client.min'
+  }
+});
+
+requirejs(['jquery', 'oc-client.min'], ($, ocClient) => {
```

Comment:
this oc-client.min is the one I've added in the folder below (not the one from the chegg cdn)